### DISCUSSION TITLE
为何时间间隔这么久

### DISCUSSION THREAD

**Discussion by zzwwr** [PaddlePaddle/PaddleOCR#12444] [Q&A]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/83124400/626112c3-12ce-473d-8e09-be55ef07032d)
我在调用paddleocr的时候 us_gpu=true 可是后面的结果显示读取到识别这段过程足足等待时间有5秒！请问这是什么问题
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/83124400/8fad49d5-a894-44f6-b2ab-bd786886dba8)

**Comment by invokerbyxv** [comment]
5秒是加载模型的时间。实际部署时，加载一次后，后续识别可重复使用ocr实例。识别花费的时间参考日志中的elapsed。
